client/pages/forms/index.js

```diff
@@ -1,5 +1,5 @@
 import Image from "next/image";
-import Social from "G:/NFT MarketPlace Project Hackathon/NFT-Marketplace---Build-It-Lumos/client/Components/.jsx";
+import Social from "G:/NFT MarketPlace Project Hackathon/NFT-Marketplace---Build-It-Lumos/client/Components/Social.jsx";
```

Comment:
Aise import nahi karte!!

It should be 
```sh
import Social from "../Components/Social.jsx";
```

and ye Maine kal fix kardiya, so no need!!

Reply:
Are pata he bhai, path show hi nahi ho raha tha to relative path ki jagah absolute path daal diya 
isme bhi kuch galat nahi he kar sakte he

Reply:
But humlog ke local pc mein G folder mein nhi hai na, so error de rha tha